websites/M/MyTF1/presence.ts

```diff
@@ -0,0 +1,168 @@
+		presenceData.state = document.querySelector(".VideoSummary_title_o8ZzQ").textContent;
+		presenceData.smallImageKey = "live";
+		presenceData.smallImageText = live;
+		presenceData.largeImageKey = "mytf1";
+		presenceData.buttons = [{ label: "View Page", url: window.location.href }];
+
+	} else if (path.includes("/programmes-tv")) {
+		presenceData.details = "Programmes TV";
+		presenceData.state = browse + " Liste des programmes";
+		delete presenceData.smallImageKey;
+		delete presenceData.smallImageText;
+		if (path.split("/")[1] == "programmes-tv") {
+			presenceData.largeImageKey = "mytf1";
+		} else {
+			presenceData.largeImageKey = path.split("/")[1];
+		}
+		presenceData.buttons = [{ label: "View Page", url: window.location.href }];
+		
+	} else if (path.startsWith("/mon-compte")) {
+		presenceData.details = viewProfile;
+		presenceData.state = browse;
+		delete presenceData.smallImageKey;
+		delete presenceData.smallImageText;
+		presenceData.largeImageKey = "mytf1";
+		delete presenceData.buttons;
+		 
+	} else if (path.split("/")[3].includes("videos") && path.split("/")[4].includes("-")) {
+		presenceData.details = document.querySelector(".VideoSummary_programName_xXfYS").textContent;
+		presenceData.state = document.querySelector(".VideoSummary_title_o8ZzQ").textContent;
+		presenceData.smallImageKey = "replay";
+		presenceData.smallImageText = watchingVid;
+		presenceData.largeImageKey = path.split("/")[1];
+		presenceData.buttons = [{ label: "View Page", url: window.location.href }];
+	} else if (path.split("/")[2].includes("-")) {
+		presenceData.details = document.querySelector(".Tabs_tabs__list_FwTdR").querySelector("h1").textContent;
+		presenceData.state = browse + " Liste des replays";
+		delete presenceData.smallImageKey;
+		delete presenceData.smallImageText;
+		presenceData.buttons = [{ label: "Voir les replays", url: window.location.href }];
+		if(presenceData.details == "Coupe du Monde de la FIFA 2022") {
+			presenceData.largeImageKey = "fifa";
+		} else {
+			presenceData.largeImageKey = path.split("/")[1];
+		}
+		
+	} else if (path.split("/")[3].includes("videos")) {
+		presenceData.details = document.querySelector(".ProgramPageVideo_paddingMenu_cEuG_").querySelector("h1").textContent;
+		presenceData.state = browse + " Liste des vidéos";
+		delete presenceData.smallImageKey;
+		delete presenceData.smallImageText;
+		presenceData.buttons = [{ label: "View Page", url: window.location.href }];
+		if(presenceData.details == "Coupe du Monde de la FIFA 2022") {
+			presenceData.largeImageKey = "fifa";
+		} else {
+			presenceData.largeImageKey = path.split("/")[1];
+		}
+		
+	} else {	
+		presenceData.details = browse;
+		delete presenceData.state;
+		delete presenceData.smallImageKey;
+		delete presenceData.smallImageText;
+		presenceData.largeImageKey = "mytf1";
+		presenceData.buttons = [{ label: "View Page", url: window.location.href }];
```

Comment:
This [button](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) has the possibility of redirecting to the main page, which is not allowed.
```suggestion
```